crates/bevy_reflect/src/lib.rs

```diff
@@ -772,18 +772,20 @@ mod tests {
             foo: String,
 
             // Use `get_bar_default()`
-            #[reflect(default = "get_bar_default")]
-            #[reflect(ignore)]
-            bar: usize,
```

Comment:
is there still a test checking that it works if each is on its own line?

Reply:
Good point. I'll include another field here to test that.